Solutions/CohesitySecurity/Data Connectors/Helios2Sentinel/zip.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/zsh
+SCRIPT=$(realpath "$0")
```

Comment:
Please add a clear description on the top of the file about what this file is supposed to do and where's it's supposed to be used and if there are any input parameters.

Reply:
no input needed, and you could run it from any where..
it will cd to where this script file located.